infrastructure/src/test/java/org/corfudb/infrastructure/orchestrator/OrchestratorTest.java

```diff
@@ -148,7 +147,7 @@ private void sendAndValidateWorkflowDispatch(RequestPayloadMsg payload, UUID exp
      */
     @Before
     public void setup() {
-        mockServerContext = mock(ServerContext.class);
+        ServerContext mockServerContext = mock(ServerContext.class);
```

Comment:
this change will affect the test and most likely breaks some of them, please rollback

Reply:
Could you please help to understand which ones will break? 
Based on what I inspected, `mockServerContext` was only used in the `setup()` method to create the `orchestrator`, and only this orchestrator is being used everywhere else in the tests. No unit tests are failing as of now.

Reply:
Oh I see, mockServerContext not used, then everything is ok